source/extensions/all_extensions.bzl

```diff
@@ -28,3 +28,13 @@ def envoy_windows_extensions():
         windows_extensions.append(path)
 
     return windows_extensions
+
+def envoy_ppc_extensions():
+   all_extensions = envoy_all_extensions()
+   luaijt_path = ''
+   for extension_path in all_extensions:
+      if extension_path.find("lua") > 0:
+         luajit_path = extension_path
+   if luajit_path != '':
+      all_extensions.remove(luajit_path)
```

Comment:
i cleaned that up a bit. since i'm still a bazel novice, i'm stuck on how to conditionally configure the skip_targets for envoy_dependencies, though. right now it's a check for the config_setting, but, that's a text string at that point. :) is there some way to query the target cpu that isn't a config setting?

Reply:
yah, i had played with select(), but you can't put a select statement into anything called from a workspace file, so, i left the text string in for now as I'm stumped. If @jmillikin-stripe has a solution that would be awesome.